src/module/chat/chat-message-types.ts

```diff
@@ -0,0 +1,34 @@
+import type { ComputedTableType } from '../roll-table/roll-table-types'
```

Comment:
i'm not sure myself. the real reason a bunch of type-only files are `*.ts` is so that i get static type checking from vscode when writing them.

there's probably value in the convention, if there is one. i could monkey with `tsconfig.json` to check `*.d.ts` files (though this might make some declarations *more* verbose)... but as type imports they won't affect the compiled source, so i didn't really pursue it.